workers/unity/Assets/Gdk/Core/Error/LoggingDispatcher.cs

```diff
@@ -0,0 +1,30 @@
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Core
+{
+    public enum LogLevel
```

Comment:
Why do we need yet another Log level enum? Unity already has https://docs.unity3d.com/ScriptReference/LogType.html

Reply:
Possible reason for another type - the number of SpatialOS log levels is larger than the Unity one. I don't think it's a massive problem if we only expose log, error, and warning though.

Other possible reason is that the logger is slightly more general than what Unity has. A user could want to handle fatal errors for example.